internal/backend/grpc/management.go

```diff
@@ -621,6 +623,36 @@ rolloutLoop:
 	return s.triggerManifestResync(ctx, requestContext)
 }
 
+// ReadAuditLog reads the audit log from the backend.
+func (s *managementServer) ReadAuditLog(_ *emptypb.Empty, srv grpc.ServerStreamingServer[management.ReadAuditLogResponse]) error {
+	rdr, err := s.auditor.ReadAuditLog()
```

Comment:
I think we should enforce Admin role here:
```
	if _, err := s.authCheckGRPC(src.Context(), auth.WithRole(role.Admin)); err != nil {
		return err
	}
```

Reply:
👍